task.c

```diff
@@ -94,6 +87,54 @@ static int iio_task_run(void *d)
 
 	return 0;
 }
+static int __iio_task_sync(struct iio_task_token *token, unsigned int timeout_ms, bool token_destroy)
```

Comment:
I believe it's best to avoid using a double underscore at the beginning of an identifier name, because it is reserved for system use.

Reply:
Yeah, I can change that. Though I find it very unlikely to have any kind of conflict with the above symbol :)